runtime/etiao/src/main/java/quarks/runtime/etiao/EtiaoJob.java

```diff
@@ -51,6 +53,10 @@ public EtiaoJob(DirectGraph graph, String topologyName, ServiceContainer contain
         ControlService cs = container.getService(ControlService.class);
         if (cs != null)
             cs.registerControl(JobMXBean.TYPE, getId(), getName(), JobMXBean.class, new EtiaoJobBean(this));
+        
+        this.jobs = (JobRegistry) container.getService(JobRegistryService.class);
```

Comment:
Thanks for catching this.  My initial thinking was that a JobRegistryService provides restricted functionality to the application code: query, subscribe to job changes, and remove jobs, while **adding** jobs to the registry is the responsibility of the runtime, which would directly call the service implementation. A job is registered with a CONSTRUCTED state when a DirectTopology is created, then it changes to RUNNING when the topology is successfully submitted.  Allowing application code to register jobs might clash with the current implementation if applications add jobs already registered by the runtime.

Currently the JobRegistryService docs indicate "registration" of Jobs as well, but there is no add() method, which is confusing. Which of the following is preferable?
1. JobRegistryService restricted to job query / removal (I'll update the javadocs accordingly.)
2. Add registration to JobRegistryService.